src/components/Tailwind/Panels/GeoBlocked.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import PageWarning from '../Layout/PageWarning'
+import { GeofenceCountry, useGeofence } from '../../../halo-hooks/useGeofence'
+
+interface GeoBlockedProps {
+  children: React.ReactNode
+  country: GeofenceCountry
```

Comment:
I think we don't need the calling component to specify this.

We can have a hardcoded blocked country list private on this component instead, e.g.

```
const BLOCKED_COUNTRIES = [ GeofenceCountry.SINGAPORE ];
```